File: utils/scoreCalculator.js
function calculateScore(savings,investments,debt){

return savings + investments - debt

}

function getTitle(score,savings,investments,debt){

if(investments > savings && investments > debt)
return "The Investor"

if(debt > savings)
return "The Risk Taker"

if(savings > investments)
return "The Saver"

return "The Spender"

}

File: utils/score_calculator.py
def calculate_final_score(savings, debt, investments, score):

    final_score = savings + (investments * 1.5) - debt + score

    return final_score


def evaluate_score(score):

    if score < 5000:
        return "Poor financial decisions"

    elif score <= 10000:
        return "Average financial management"

    else:
        return "Strong financial management"